lib/ansible/plugins/filter/core.py

```diff
@@ -199,8 +199,11 @@ def from_yaml(data):
     return data
 
 @environmentfilter
-def rand(environment, end, start=None, step=None):
-    r = SystemRandom()
+def rand(environment, end, start=None, step=None, seed=None):
+    if seed is None:
+        r = SystemRandom()
+    else:
+        r = Random(seed)
```

Comment:
Meh, it even is documented that SystemRandom ignores the seed. Sorry for the noise.